colaraz/lms/static/js/discussion/colaraz_discussion_utils.js

```diff
@@ -0,0 +1,57 @@
+$(document).on("click", "button.action-vote", function(event) {
+    var currentElement = this;
+    if ($(currentElement).closest("article.discussion-article").length <= 0) {
+      return;
+    }
+
+    var is_voting_disabled = $(currentElement)
+                              .closest("li.actions-item")
+                              .prop("className")
+                              .split(" ")
+                              .includes("is-disabled");
+
+    if (is_voting_disabled) {
+      console.log("Voting is disabled. An author can't vote his own post or comment.");
+      return;
+    }
+
+    var voteType = "";
+    if ($(this).attr("aria-checked") == "true") {
+      voteType = "unvote";
+    } else {
+      voteType = "upvote";
+    }
+    castVote(this, voteType);
+});
+
+function castVote(currentElement, voteType) {
+    var threadID = $(currentElement).closest("article.discussion-article").attr("data-id")
+    var threadVoteUrl = "courses/${course.id}/discussion/threads/" + threadID + "/" + voteType + "?ajax=1";
+
+    $.ajax({
+        type: "POST",
+        url: threadVoteUrl,
+        success: function (data) {
+            if (voteType == "upvote") {
+                $(currentElement).attr("aria-checked", "true");
+            } else {
+                $(currentElement).attr("aria-checked", "false");
+            }
+
+            var outputString = "";
+            if (data.votes.up_count != 1) {
+                outputString = data.votes.up_count + " Votes";
```

Comment:
I think other files are using signle quotes. Shouldn't we use that too? what do u think?